test/searchModal/mocks/mocks.js

```diff
@@ -0,0 +1,37 @@
+const mockedResults = {
```

Comment:
oO :o

Please not use global variables, even in unit tests!

Here you can either use a JSON file, or export your resources from a regular AMD module.